From 3badd79adc9197a34d72cf0c1bd1f0c8ba1d51ae Mon Sep 17 00:00:00 2001
From: Max-Julian Pogner <max-julian@pogner.at>
Date: Sun, 15 May 2022 10:49:25 +0200
Subject: [PATCH 2/2] Dh_Lib.pm: prefer `return` over `return undef`

They behave differently depending on the context (e.g. list, scalar, etc)
the sub is called in.
---
 lib/Debian/Debhelper/Dh_Lib.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Debian/Debhelper/Dh_Lib.pm b/lib/Debian/Debhelper/Dh_Lib.pm
index be79befa..41bc76f5 100644
--- a/lib/Debian/Debhelper/Dh_Lib.pm
+++ b/lib/Debian/Debhelper/Dh_Lib.pm
@@ -1764,7 +1764,7 @@ sub getpackages {
 
 sub _strip_spaces {
 	my ($v) = @_;
-	return undef if not defined($v);
+	return if not defined($v);
 	$v =~ s/^\s++//;
 	$v =~ s/\s++$//;
 	return $v;
-- 
2.35.2

